include/envoy/server/fatal_action_config.h

```diff
@@ -18,10 +19,9 @@ class FatalAction {
   virtual ~FatalAction() = default;
   /**
    * Callback function to run when we are crashing.
-   * @param current_object the object we were working on when we started
-   * crashing.
+   * @param objects a vector of objects we were working on when we started crashing.
    */
-  virtual void run(const ScopeTrackedObject* current_object) PURE;
+  virtual void run(const std::vector<const ScopeTrackedObject*>& tracked_objects) PURE;
```

Comment:
virtual void run(const absl::Span<const ScopeTrackedObject*>& tracked_objects) PURE;

To avoid tieing us to a vector implementation.